src/platform/datapath_winkernel.c

```diff
@@ -2655,7 +2655,7 @@ CxPlatSendDataAllocSegmentBuffer(
         return (QUIC_BUFFER*)&SendData->ClientBuffer;
     }
 
-    Buffer = CxPlatSendDataAllocDataBuffer(SendData, &ProcContext->LargeSendBufferPool);
+    UINT8* Buffer = CxPlatSendDataAllocDataBuffer(SendData, &SendData->Owner->LargeSendBufferPool);
```

Comment:
Is the double indirection going to have a perf impact?

Reply:
No. It already was doing this, but I just removed the temporary, local variable.